include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -1735,7 +1741,12 @@ oneapi::dpl::__internal::__enable_if_host_execution_policy<_ExecutionPolicy, _Ou
 __pattern_reverse_copy(_ExecutionPolicy&&, _BidirectionalIterator __first, _BidirectionalIterator __last,
                        _OutputIterator __d_first, _IsVector __is_vector, /*is_parallel=*/::std::false_type) noexcept
 {
-    return __internal::__brick_reverse_copy(__first, __last, __d_first, __is_vector);
+    if (__first != __last)
```

Comment:
We dont need such check here. We do such trivial check just in parallel patterns.

Reply:
Fixed